content/02.introduction.md

```diff
@@ -1,7 +1,9 @@
+Ancient DNA (aDNA) has become a widely accepted source of biological data, helping to provide new perspective for a range of fields including archaeology, ecology, cultural heritage, and palaeontology. The utilisation of next-generation-sequencing has allowed the recovery of aDNA from a wide variety of sources, including but not limited to, the skeletal remains of animals [@doi:10.1016/j.cub.2015.04.007, @doi:10.1038/nature12323, @doi:10.1073/pnas.1901169116, @doi:10.1073/pnas.1710186114], modern and archaic humans [@doi:10.1038/s41586-018-0094-2, @doi:10.1126/science.1224344, @doi:10.1038/s41586-018-0455-x], bacteria [@doi:10.1038/nature13591, @doi:10.1073/pnas.1812865115, @doi:10.1371/journal.ppat.1006997], viruses [@doi:10.1038/s41586-018-0097-z, @doi:10.7554/eLife.36666], plants [@doi:10.1111/eva.12594, @doi:10.1038/s41559-019-0921-3], coprolites [@doi:10.1016/j.chom.2019.08.018, @doi:10.7717/peerj.9001], dental calculus [@doi:10.1038/ng.2906, @doi:10.1038/nature21674], sediments [@doi:10.1038/nature12921, @doi:10.1126/science.aam9695], medical slides [@doi:10.1093/molbev/msz264], parchment [@doi:10.1098/rstb.2013.0379], and most recently ancient ‘chewing gum’ [@doi:10.1038/s41467-019-13549-9, @doi:10.1038/s42003-019-0399-1]. Improvement in laboratory protocols to increase yields of otherwise trace amount of DNA has at the same time led to studies that can total hundreds of ancient individuals [@doi:10.1038/nature25738, @doi:10.1038/nature25778], spanning single [@doi:10.1038/nature10549] to thousands of organisms [@doi:10.1038/ng.2906]. These differences of disciplines have led to a heterogeneous landscape in terms of types of analyses, and thus various types of computing resources required by different labs. Particularly, this is true for regions of the world where aDNA as a field of research is still emerging.
```

Comment:
> Particularly, this is true for regions of the world where aDNA as a field of research is still emerging.

This is particularly,  true for regions of the world where aDNA as a field of research is still emerging.

But do you really want to have this sentence ?